### ISSUE TITLE
Can't Set Focus, ListBox Is Empty

### ISSUE THREAD

**Issue by weilbith** [Nanoseb/ncTelegram#39]
After successfuly initialize `telegram-cli` and trying to start `nctelegram` now, lead to the following error message:
```python
bind: Address already in use
Traceback (most recent call last):
  File "/usr/bin/nctelegram", line 93, in <module>
    ncTelegram.Telegram_ui(config_full)
  File "/usr/lib/python3.7/site-packages/ncTelegram/__init__.py", line 73, in __init__
    self.msg_widget = MessageWidget(self)
  File "/usr/lib/python3.7/site-packages/ncTelegram/ui_msgwidget.py", line 25, in __init__
    self.get_history()
  File "/usr/lib/python3.7/site-packages/ncTelegram/ui_msgwidget.py", line 66, in get_history
    self.draw_separator()
  File "/usr/lib/python3.7/site-packages/ncTelegram/ui_msgwidget.py", line 246, in draw_separator
    self.focus_position = self.separator_pos
  File "/usr/lib/python3.7/site-packages/urwid/listbox.py", line 578, in set_focus
    raise IndexError("Can't set focus, ListBox is empty")
IndexError: Can't set focus, ListBox is empty
```

I've installed it through the ArchLinux user repository.

**Comment by weilbith** [comment]
Definitely not. I already used it and interacting with `telegram-cli` directly seems works fine.

**Comment by jonburga** [comment]
I have the same issue. Have used both telegram-desktop and telegram-cli successfully

**Comment by jonburga** [comment]
In particular, this happens only with urwid 2.0. Downgrading 2.0 to 1.3 yields "module urwid cannot be found".

**Comment by Mikepicker** [comment]
I created a pull request which fixes this issue, it is referenced here: #40 
Please, test it and let me know if it works for you.

**Comment by Nanoseb** [comment]
Thanks again @Mikepicker ! closing this now, it seems to work for everyone
